### DISCUSSION TITLE
Benefits of using frontmatter in Obsidian

### DISCUSSION THREAD

**Discussion by danichu1** [BryanJenksCommunity/FAQ#189] [Q&A] [answered]
Hello everyone!

I watched a couple videos that talk about how you can use notes as tags, a method that I really like. I also learned about frontmatter. And in trying to combine the two I've stumbled upon a problem. When I use frontmatter in a template to create room for my tags, it offers only space for one note-tag. My questions are the following:
1) Is it possible to link to multiple notes in one line of frontmatter?
2) Is there an added benefit of using frontmatter besides differentiating between the note and metadata? I'm not very familiar with vault organisation.

The videos I watched didn't talk about frontmatter. From Bryan Jenks I watched a video from three years ago (https://www.youtube.com/watch?v=zIh1S7ra3aI). He didn't seem to use frontmatter yet.

A helpful answer would be much appreciated!

**Comment by ptsouchlos** [comment]
Hello!

Yes you should be able to link multiple notes in front matter. To clarify, when you say tags do you mean `#tag` or `[[tag]]`. In either case you can have multiple tags in front matter though I'm not sure if they'll show up in "one line" but this should be possible.

 On using front matter versus not, this is a long standing debate in the Obsidian community I feel like. I think an advantage of _not_ using front matter is you can reserve it for true metadata instead of relying on it to create connections to other notes. Additionally, there are some things you can do with the data view plugin if you don't use front matter. These points were not as important to me so I am currently using frontmatter in all my notes in my Obsidian vault. 

Hope this helps.

**Reply by danichu1** [reply]
Thanks for the answer. 

By tag I meant [[tag]]. Could you tell me how to list multiple link tags in front matter? When I list two links Obsidian fuses them into one link that I didn't mean to create.

**Reply by ptsouchlos** [reply]
Sure thing.

How are you editing your files? Source mode? Edit mode?

**Reply by ptsouchlos** [reply]
If you have the front matter type selected as `list` you should be able to have multiple links:

![image](https://github.com/BryanJenksCommunity/FAQ/assets/6591180/bc6ca034-de02-4903-b9e1-84dcd257faad)
![image](https://github.com/BryanJenksCommunity/FAQ/assets/6591180/26e852a6-910c-47b1-9918-bab3ba42a0d8)

**Reply by danichu1** [reply]
I didn't think of checking that. Thank you!

**Comment by tallguyjenks** [comment, marked as answer]
> Is it possible to link to multiple notes in one line of frontmatter?

Yes, example:

```yaml
---
tags:
  - 🌱️
publish: true
aliases: []
created: 2023-08-31 0710
updated: 2023-09-07 1615
anatomy:
  - "[[note 1]]"
  - "[[note 2]]"
---
```

you CAN write a yaml array for the note syntax above but obsidian automatically uses the bullet list array notation if you switch to preview mode but ultimately it doesn't matter.

> Is there an added benefit of using frontmatter besides differentiating between the note and metadata? I'm not very familiar with vault organisation.

yes, dataweave queries for example

**Reply by danichu1** [reply]
Though it took me a while to resolve my confusion, your anwer was helpful. I now know the difference between source mode and preview mode, and how they work. As you can see, I truly have a lot to learn. I still don't know that much about the advantages of frontmatter, but I will keep using it. 
Thanks for the anwer!

**Comment by danichu1** [comment]
My last questions @tallguyjenks.  

I see you're using a modification timestamp. Does it update automatically? And if so, how?
Do I get it correctly that you need to use the word 'tags' before the status symbol to make it visible in the graph?

**Reply by tallguyjenks** [reply]
Not sure what you mean by "tags before status symbol" 

Timestamp is automatic from the "update time on edit" plugin

**Reply by danichu1** [reply]
I meant this part of the metadata you sent me

tags:
  - 🌱️
  
A hashtag used in frontmatter only shows up in my graph if it is given the tag property, which seems impossible for Obsidian if I don't precede it with 'tags'. 
I realise it might be possible to stay organized without it being listed as a tag but I was wondering what you had to say about it.

**Reply by tallguyjenks** [reply]
yaml metadata is key value pairs, for the note to have a tag applied to it via the yaml it needs the syntax displayed above

```yaml
tags:
  - 🌱️
```
